src/services/licenseService.ts

```diff
@@ -0,0 +1,18 @@
+export interface License {
+  licenseId: string;
+  name: string;
+}
+
+export const fetchLicenses = async (): Promise<License[]> => {
+  try {
+    const response = await fetch("https://raw.githubusercontent.com/spdx/license-list-data/main/json/licenses.json");
```

Comment:
can you move this over to a constants file?

Reply:
i just meant the url, not the whole thing

Reply:
Moved the license url to constant file